ray_on_golem/client/client.py

```diff
@@ -171,16 +171,24 @@ def shutdown_webserver(self) -> models.ShutdownState:
 
         return response.shutdown_state
 
-    def is_webserver_running(self) -> bool:
+    def get_webserver_state(self) -> Optional[bool]:
```

Comment:
hmm... given that this method still returns a plain `bool`, I'd leave it as it was ... as it does _not_ return the webserver state... 

on the other hand, it could be renamed to e.g. `is_webserver_serviceable` ...

```suggestion
    def is_webserver_serviceable(self) -> Optional[bool]:
```